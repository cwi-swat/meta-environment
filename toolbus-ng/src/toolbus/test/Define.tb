#define S "string"
#define I identifier
#define T1 and(true,false)
#define T2 and("a", "b")
#define T3 and(T1, T2)

process Q is
  printf("S = %t\n", S) .
  printf("I = %t\n", I) .
  printf("T1 = %t\n", T1) .
  printf("T2 = %t\n", T2) .
  printf("T3 = %t\n", T3)

toolbus(Q)


