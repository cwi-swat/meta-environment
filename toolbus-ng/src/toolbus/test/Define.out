S = "string"
I = identifier
T1 = and(true,false)
T2 = and("a","b")
T3 = and(and(true,false),and("a","b"))
